### PR TITLE
Monitoring

### PR DESCRIPTION
**Pull request by anton-okolelov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,26 @@
 DBMeter [![Build Status](https://travis-ci.org/DBMeter/DBMeter.svg?branch=master)](https://travis-ci.org/DBMeter/DBMeter)
 =============
+
```
Comment (hack3p) [inline README.md:3]: We can extract this manual in shell script. It is would be easyer for frequent install. Is'n it?
> sh install.sh

File: src/postgres_monitoring/mod.rs [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    loop {
+        thread::sleep(Duration::from_secs(interval));
+
+        let rows = &connection
+            .query(
+                "SELECT queryid, query, calls, total_time \
+                 FROM dbmeter.pg_stat_statements()",
+                &[],
+            )
+            .expect("ошибка");
```
Comment (hack3p) [inline src/postgres_monitoring/mod.rs]: Detailed english error message is needed

File: src/postgres_monitoring/pg_history.rs [hunk 3]
```diff
@@ -0,0 +1,33 @@
+use postgres_monitoring::pg_stat::PgStat;
+use std::collections::HashMap;
+use postgres_monitoring::monitoring_output::MonitoringOutput;
+
+pub struct PgHistory {
```
Comment (hack3p) [inline src/postgres_monitoring/pg_history.rs]: it seems to me, pg prefix is redundant, because we are in postgres namespace

**Review by hack3p — CHANGES_REQUESTED** [review]

**Comment by anton-okolelov** [comment]
Some fixes has been made

File: README.md (hunk 1 above)
Comment (anton-okolelov) [reply README.md:3]: Good idea
But I believe for the moment there is almost nothing to automate (just one line of code) and it still will require manual steps (like set other password )

**Review by hack3p — APPROVED** [review]

**Pull request merged**